pypeit/spectrographs/keck_hires.py

```diff
@@ -208,6 +208,26 @@ def compound_meta(self, headarr, meta_key):
             msgs.error("Not ready for this compound meta")
 
 
+    def config_independent_frames(self):
+        """
+        Define frame types that are independent of the fully defined
+        instrument configuration.
+
+        This method returns a dictionary where the keys of the dictionary are
+        the list of configuration-independent frame types. The value of each
+        dictionary element can be set to one or more metadata keys that can
+        be used to assign each frame type to a given configuration group. See
+        :func:`~pypeit.metadata.PypeItMetaData.set_configurations` and how it
+        interprets the dictionary values, which can be None.
+
+        Returns:
+            :obj:`dict`: Dictionary where the keys are the frame types that
+            are configuration-independent and the values are the metadata
+            keywords that can be used to assign the frames to a configuration
+            group.
+        """
+        return {'bias': 'binning', 'dark': 'binning'}
```

Comment:
`binning` isn't one of the configuration keys for HIRES.  It doesn't need to be for its use here to associate biases and darks with specific configurations, but should we add it to `configuration_keys`?

Reply:
Yes, I believe so... I have added it.

Reply:
I guess you can remove `config_independent_frames` for hires since it's equal to the default.

Reply:
Yes - that's a good point - done!